docs/en/advanced-features/plugin-exclusion.md

```diff
@@ -0,0 +1,19 @@
+# Plugin Exclusion
+
+The plugin exclusion is used during the **compilation phase** to exclude specific plugins, through their names.
+Consequently, the codes of these excluded plugins will not be weaved in, then, no relative tracing and metrics.
+
+## Configuration
+
+
+```yaml
+plugin:
+  # List the names of excluded plugins, multiple plugin names should be splitted by ","
+  # NOTE: This parameter only takes effect during the compilation phase.
+  excluded: ${SW_AGENT_PLUGIN_EXCLUDES:}
```

Comment:
@liuhaoyang I added this, and update the config name(excludes -> excluded), as well as comments. Please update code accordingly.

Reply:
fixed.